.changeset/polite-apples-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-css": patch
```

Comment:
You would also need to include `@itwin/itwinui-react` in the changeset since although these are only css changes, `itwinui-react` is also affected. Another reason is that `itwinui-react` does not have a dependency on `itwinui-css`. So from a user's perspective, only `itwinui-react` will seem changed, and thus we need a `itwinui-react` changeset.

```suggestion
"@itwin/itwinui-css": patch
"@itwin/itwinui-react": patch
```

Reply:
Done in latest commit.  ([63ea24c](https://github.com/iTwin/iTwinUI/pull/1884/commits/63ea24c32ddebf81286072efc760a2f02f6bdfc6))